src/App.js

```diff
@@ -0,0 +1,20 @@
+import { Routes, Route } from 'react-router-dom';
+import Home from './modules/Home/Home';
+import Navbar from './modules/Navbar/Navbar';
+import Countries from './modules/Countries/Countries';
+import Details from './modules/Details/Details';
+
+function App() {
```

Comment:
- [ ] As requested by the previous reviewer, please kindly make your homepage categories filterable.  This could be a search box or a drop-down list.  

## Explanation

This is so you can show off and practice your redux skills in managing the state of the rendering.  It is also a project requirement for this particular project:

![image](https://user-images.githubusercontent.com/57421823/157658111-06e0a368-5189-47ac-b8b2-faddf6ceb49a.png)


## Hints



That way, we will meet this particular requirement 💪

Reply:
Hi @Meltrust , that requirement is met already. I filtered by continents, it only appears the countries for each continent. I filter them on the reducer. And the reviewer didn't request any other changes besides the acknowledgments part. Am I missing something?

Reply:
"When the page loads, the webapp shows the list of categories that could be filtered out by some parameter"
I made that, by continents, the categories are the continents and when you click on the category(Continent) it filters

Reply:
Hi @guerreiropedr0,


I'm just making reference to the last reviewer's request which you maybe didn't notice:

![image](https://user-images.githubusercontent.com/57421823/157661414-c77f813e-ef1e-4873-8c32-2ca8d838a72f.png)

...From which sticking strictly to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/157663151-980c6cb3-ac55-475a-afd6-c251faaca052.png)

...and

![image](https://user-images.githubusercontent.com/57421823/157663207-e5492e01-6cf6-46cc-a449-4f6f0357631e.png)

Nevertheless, if you don't agree with our reviews please feel free to ask for a second opinion by filling out this [form](https://airtable.com/shrQAqnBwek5a0O0s).

Thank you for your kind feedback.

Reply:
But am I not filtering in the reducer? I'm making the search better by not displaying every country in the world, but by selecting the continent

Reply:
![image](https://user-images.githubusercontent.com/78462401/157667687-b1469bed-d0fb-4c32-85f3-643dbeb392df.png)
In this image, it says some parameter, I chose the continents

Reply:
If your parameter is the continents as you kindly say, that would certainly mean you are not listing the categories on the homepage, as the same requirement phrase asks for, and instead, you are linking to them.

I hope this resolves your inquiry.

Reply:
I fetch 249 countries and I have a list of 249 countries in my state, in the home page I'm not showing the whole list of countries, I'm showing 6 regions, which means I'm filtering the 249 countries by their region.

Reply:
I have filled out the form, I'm sorry this didn't work
Have a nice day and happy coding.